src/graphql/types/Book.ts

```diff
@@ -0,0 +1,21 @@
+//TODO: change to book detail information
+
+import { objectType } from 'nexus';
+
+const Book = objectType({
+  name: 'Book',
+  description: 'The details of a book.',
+  definition(t) {
+    t.id('id');
+    t.string('isbn');
+    t.id('author_id');
```

Comment:
Can you add a TODO here to add the correct field for author with the subtype resolver?